columnflow/calibration/jets.py

```diff
@@ -173,11 +210,29 @@ def jec(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
         Rho=events.fixedGridRhoFastjetAll,
     )
 
+    # store pt and phi of the full jet system
+    jetsum = ak.sum(events.Jet, axis=1)
```

Comment:
AFAIK there is a pT threshold on the jets that enter the Type-I MET correction (maybe 15 GeV?), otherwise one could introduce a bias on the MET. Any plans to implement this? (Should be enough to just mask/filter the jets in the sum here based on some configurable threshold.)

Reply:
Yep, I'm going to add the two cuts mentioned in the [https://twiki.cern.ch/twiki/bin/view/CMS/JECAnalysesRecommendations?rev=19#Minimum_jet_selection_cuts](JECAnalysesRecommendations).

Reply:
Added :)